requirements.txt

```diff
@@ -15,3 +15,4 @@ fastapi
 sse-starlette
 matplotlib
 fire
+badam
```

Comment:
badam requires `torch>2.1.0`, but we want to make llama factory usable in torch lower version, so I suggest adding this package to extra requirements instead of core requirements like galore: https://github.com/hiyouga/LLaMA-Factory/blob/7dc72fb58cb988418323f63821a21a184ecf0718/setup.py#L26

Reply:
Thanks for your suggestion, i have change the file accordingly.